cmd/podman/machine/list.go

```diff
@@ -79,12 +97,8 @@ func list(cmd *cobra.Command, args []string) error {
 		return listResponse[i].Running
 	})
 
-	defaultCon := ""
-	con, err := registry.PodmanConfig().ContainersConfDefaultsRO.GetConnection("", true)
-	if err == nil {
-		// ignore the error here we only want to know if we have a default connection to show it in list
-		defaultCon = con.Name
-	}
+	// ignore the error here we only want to know if we have a default connection to show it in list
+	defaultCon, _ := registry.PodmanConfig().ContainersConfDefaultsRO.GetConnection("", true)
```

Comment:
This will panic in case of errors because it return nil so you deference a nil pointer later